homeassistant/components/jvc_projector/sensor.py

```diff
@@ -0,0 +1,85 @@
+"""Sensor platform for JVC Projector integration."""
+
+from __future__ import annotations
+
+from jvcprojector import const
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import JvcProjectorDataUpdateCoordinator
+from .const import DOMAIN
+from .entity import JvcProjectorEntity
+
+JVC_SENSORS = (
+    SensorEntityDescription(
+        key="power",
+        translation_key="jvc_power_status",
+        device_class=SensorDeviceClass.ENUM,
+        icon="mdi:power",
+        options=[
+            const.OFF,
+            const.STANDBY,
+            const.ON,
+            const.WARMING,
+            const.COOLING,
+            const.ERROR,
+        ],
+    ),
+    SensorEntityDescription(
+        key="input",
+        translation_key="jvc_input",
+        device_class=SensorDeviceClass.ENUM,
+        icon="mdi:video-input-hdmi",
+        options=[
+            "hdmi1",
+            "hdmi2",
+            const.NOSIGNAL,
+        ],
+    ),
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the JVC Projector platform from a config entry."""
+    coordinator: JvcProjectorDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+
+    sensors_descriptions: list[SensorEntityDescription] = []
+    for description in JVC_SENSORS:
+        sensors_descriptions.append(description)
```

Comment:
What are you doing here

Reply:
Create the dynamic list of sensors to be added in the line after